aws/eks/outputs.tf

```diff
@@ -26,6 +26,10 @@ output "subnets" {
   value = module.eks-subnets.subnets
 }
 
+output "eks-vpc" {
+  value = module.eks-vpc
+}
+
 output "vpc_id" {
   value = module.eks-vpc.vpc_id
 }
```

Comment:
I had in my head that it would make sense to just call this output `vpc`...and also remove the output below. What do you think about those two ideas?

Reply:
I left the other output because I didn't want it to be a breaking change, but that is fine too. Also, we should use the name of the submodule in the output to not add to the confusion. (the module was originally named `vpc` and someone changed it in the past 2 months)